components/forms/StateSearch.vue

```diff
@@ -59,11 +60,25 @@ function onCloseClick() {
   search.value = ''
   emit('select', null)
 }
+
+async function focus() {
```

Comment:
Hi @mcromp, can we add a video to the description that shows how this functionality works and how the focus effect operates? Maybe we can use CSS for that animation?